da/da_test.go

```diff
@@ -29,14 +23,15 @@ import (
 const mockDaBlockTime = 100 * time.Millisecond
 
 func TestMain(m *testing.M) {
-	srv := startMockGRPCServ()
+	srv := startMockDAServ()
 	if srv == nil {
 		os.Exit(1)
 	}
 	exitCode := m.Run()
 
 	// teardown servers
-	srv.GracefulStop()
+	// nolint:errcheck,gosec
+	srv.Stop(context.TODO())
```

Comment:
We should use `Background()` here. 
TODO is only meant to be used as actual TODOs that are to be followed up and handled.

Reply:
Fixed.